tf_encrypted/protocol/pond.py

```diff
@@ -907,6 +907,12 @@ def strided_slice(self, x: "PondTensor", *args: Any, **kwargs: Any):
 
         return x_sliced
 
+    @memoize
+    def gather(
```

Comment:
how does this relate to https://github.com/tf-encrypted/tf-encrypted/pull/442?

Reply:
oh no shouldn't be there. Let me fix this.